tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -218,6 +218,25 @@ where
 		Ok(json!(json_value))
 	});
 
+	// get_mrenclave
+	let get_mrenclave_name: &str = "get_mrenclave";
```

Comment:
Hi @songzhouran I missed this small thing.
What do you think about renaming it to `state_getEnclave` 
Just to be consistent with https://github.com/litentry/litentry-parachain/blob/e27b733b84675d22438a9c4d5af0504900a540bc/tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs#L243